src/EnergyPlus/UnitarySystem.hh

```diff
@@ -451,6 +513,15 @@ namespace UnitarySystems {
 
         static void getUnitarySystemInput(EnergyPlusData &state, std::string const &Name, bool const ZoneEquipment, int const ZoneOAUnitNum);
 
+        void processInputSpec(EnergyPlusData &state,
+                              const UnitarySysInputSpec &input_data,
+                              int sysNum,
+                              bool &errorsFound,
+                              bool const ZoneEquipment,
+                              int const ZoneOAUnitNum);
+
+        //void setSystemParams(EnergyPlusData &state, Real64 &TotalFloorAreaOnAirLoop, const std::string thisObjectName);
```

Comment:
I have to figure out how to get this function back in. Temporarily commented out (and pasted the little chunk of code in 3 places) to test for diffs.